modules/installation-load-balancing-user-infra.adoc

```diff
@@ -213,10 +213,13 @@ listen api-server-6443 <1>
 listen machine-config-server-22623 <3>
   bind *:22623
   mode tcp
-  server bootstrap bootstrap.ocp4.example.com:22623 check inter 1s backup <2>
-  server master0 master0.ocp4.example.com:22623 check inter 1s
-  server master1 master1.ocp4.example.com:22623 check inter 1s
-  server master2 master2.ocp4.example.com:22623 check inter 1s
+  option  httpchk GET /readyz HTTP/1.0 <2>
```

Comment:
Should the `<2>` callout on the `option  httpchk GET /readyz HTTP/1.0` line be removed? I don't know anything about this, but it doesn't say "bootstrap" like the other lines that use that callout. I apologize if this is my ignorance talking. :)

Reply:
The <2> callout should only be on the bootstrap lines.

Reply:
Thanks, @ousleyp . Well spotted. I removed the callout reference for that line. Thanks Tim for chiming in!